Automattic/newspack-app-shell
# Handle form submission

## What it does

This introduces SPA-like (no reloading) form submissions. It was written with WP comments in mind, but should handle any `form` with an `action` attribute.

Closes #4 

## How to test

1. open a post
1. submit a valid comment
1. observe that the page does not reload, it's updated like on navigation, and the new comment is visible
1. submit an invalid comment
1. observe that the error message is prepended to the form element, in red
